server/src/test/java/org/elasticsearch/snapshots/mockstore/MockEventuallyConsistentRepository.java

```diff
@@ -215,12 +216,15 @@ public void deleteBlob(String blobName) {
             }
 
             @Override
-            public void delete() {
+            public void delete(LongConsumer resultConsumer) {
```

Comment:
I tend to think that this method could return deletion stats, WDYT?

Reply:
I tried finding an argument against doing that but couldn't :D Will refactor that :)